draft-ietf-scitt-architecture.md

```diff
@@ -656,7 +652,7 @@ There are many types of Statements (such as SBOMs, malware scans, audit reports,
 An Issuer must first decide on a suitable format (`3`: payload type) to serialize the Statement payload.
 For a software supply chain, payloads describing the software Artifacts may include:
 
-- {{COSWID}}
+- {{-COSWID}}
```

Comment:
as referenced, this doesn't render as COSWID, rather, it renders with RFC9393
Was that intentional? 

- [RFC9393]
- [CycloneDX]
- [in-toto]
- [SPDX-CBOR]
- [SPDX-JSON]
- [SLSA]
- [SWID]

Reply:
hm... good point. I can reverse the one item.

Reply:
Done